airbyte-integrations/connectors/source-kyve/source_kyve/spec.yaml

```diff
@@ -0,0 +1,43 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/kyve
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: KYVE Spec
+  type: object
+  required:
+    - pool_ids
+    - start_ids
+  properties:
```

Comment:
Can you add an order property to each of these properties in the spec?

Reply:
Done